extensions/apiconnect/src/types/oas-extensions/x-ibm-configuration/assembly/execute/set-variable.policy.ts

```diff
@@ -0,0 +1,39 @@
+export type SetVariablePolicy = DataPowerGateway | DataPowerAPIGateway;
+export type DataPowerGateway = V100;
+export type DataPowerAPIGateway = V200;
+
+export interface V100 {
+  version: '1.0.0';
+  title?: string;
+  description?: string;
+  actions:
+    | {
+        set: string;
+        value: string;
+      }
+    | {
+        add: string;
+        value: string;
+      }
+    | {
+        clear: string;
+      };
+}
+
+export interface V200 extends Omit<V100, 'version' | 'actions'> {
+  version: '2.0.0';
+  actions:
+    | {
+        set: string;
+        value: string;
+        type: 'any' | 'string' | 'number' | 'boolean'; // Documentation unclear if this is needed for `clear`.
```

Comment:
The documentation for this is unclear when `type` is needed.

Reply:
#TODO: Update `value` to allow `string | number | boolean`; Conditionally mandate `value: boolean` if `type: 'boolean'`.